src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+    private String color = getColor();
+
+    public Circle(int radius, String color) {
+        this.radius = radius;
+        this.color = color;
+    }
+
+    @Override
+    public double areaCalculator() {
+        return (radius * radius * Math.PI);
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure circle, area: " + areaCalculator()
+                + ", radius: " + radius
+                + ", color: " + color);
```

Comment:
Avoid using direct calls to 'System.out.println' in the 'draw' method. It's better to return the string representation and let the caller decide how to handle it. If you want to keep the print functionality, consider renaming the method to reflect that it's doing a print operation.